src/FlaUI.WebDriver.UITests/ElementTests.cs

```diff
@@ -9,40 +10,69 @@ namespace FlaUI.WebDriver.UITests
     [TestFixture]
     public class ElementTests
     {
-        [Test]
-        public void GetText_Label_ReturnsRenderedText()
+        [TestCase("TextBox", "Test TextBox")]
+        [TestCase("PasswordBox", "●●●●●●●●●●")]
+        [TestCase("EditableCombo", "Item 1")]
+        [TestCase("NonEditableCombo", "Item 1")]
+        [TestCase("ListBox", "ListBox Item #1")]
+        [TestCase("SimpleCheckBox", "Test Checkbox")]
+        [TestCase("ThreeStateCheckBox", "3-Way Test Checkbox")]
+        [TestCase("RadioButton1", "RadioButton1")]
+        [TestCase("RadioButton2", "RadioButton2")]
+        [TestCase("Slider", "5")]
+        [TestCase("InvokableButton", "Invoke me!")]
+        [TestCase("PopupToggleButton1", "Popup Toggle 1")]
+        [TestCase("Label", "Menu Item Checked")]
+        public void GetText_Returns_Correct_Text(string elementAccessibilityId, string expectedValue)
         {
             var driverOptions = FlaUIDriverOptions.TestApp();
             using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
-            var element = driver.FindElement(ExtendedBy.AccessibilityId("Label"));
-
+            var element = driver.FindElement(ExtendedBy.AccessibilityId(elementAccessibilityId));
             var text = element.Text;
 
-            Assert.That(text, Is.EqualTo("Menu Item Checked"));
+            Assert.That(text, Is.EqualTo(expectedValue));
         }
 
         [Test]
-        public void GetText_TextBox_ReturnsTextBoxText()
+        public void GetText_Returns_Text_For_Multiple_Selection()
         {
             var driverOptions = FlaUIDriverOptions.TestApp();
             using var driver = new RemoteWebDriver(WebDriverFixture.WebDriverUrl, driverOptions);
-            var element = driver.FindElement(ExtendedBy.AccessibilityId("TextBox"));
+            var element = driver.FindElement(ExtendedBy.AccessibilityId("ListBox"));
+
+            new Actions(driver)
+                .MoveToElement(element)
+                .Click()
+                .KeyDown(Keys.Control)
+                .KeyDown("a")
+                .KeyUp("a")
+                .KeyUp(Keys.Control)
+                .Perform();
 
             var text = element.Text;
-            
-            Assert.That(text, Is.EqualTo("Test TextBox"));
+
+            // Seems that the order in which the selected items are returned is not guaranteed.
```

Comment:
Maybe this is a FlaUI issue to report?

Reply:
It _could_ be but given that we're essentially just calling the COM pattern, I suspect it's what WPF is actually returning? WinAppDriver from what I can see though doesn't have this issue, but I'm not sure whether they're normalizing the selection themselves or not. Regardless, didn't feel like a big enough issue for now to spend much time investigating, got quite a few other PRs incoming ;)